apps/librepcb/librepcb.pro

```diff
@@ -110,8 +110,11 @@ FORMS += \
 
 # Custom compiler "lrelease" for qm generation
 isEmpty(QMAKE_LRELEASE) {
-    win32: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]\lrelease.exe
-    else: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
+    contains(QMAKE_HOST.os,"Windows"){ 
```

Comment:
I'm not sure, but maybe we could even completely remove the windows specific configuration of `QMAKE_LRELEASE`, because under Windows the `.exe` suffix is often not required. So this may even work for both Windows and Linux:

```
isEmpty(QMAKE_LRELEASE): QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
```

Could you maybe try this? Travis and AppVeyor can tell us if it breaks something, but they don't check if it also works with cross-compiling...